WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -2786,7 +2789,7 @@ static GSourceFuncs _handlerIntervention =
         {
             webkit_web_context_set_web_extensions_directory(context, browser->_extensionPath.c_str());
             // FIX it
-            GVariant* data = g_variant_new("(smsb)", std::to_string(browser->_guid).c_str(), !browser->_config.Whitelist.Value().empty() ? browser->_config.Whitelist.Value().c_str() : nullptr, browser->_config.LogToSystemConsoleEnabled.Value());
+            GVariant* data = g_variant_new("(smsbms)", std::to_string(browser->_guid).c_str(), !browser->_config.Whitelist.Value().empty() ? browser->_config.Whitelist.Value().c_str() : nullptr, browser->_config.LogToSystemConsoleEnabled.Value(), !browser->_config.MixedContentWhitelist.Value().empty() ? browser->_config.MixedContentWhitelist.Value().c_str() : nullptr);
```

Comment:
format to several lines

Reply:
functionallity was verified with other non localhost / not wildcard lists?

Reply:
Yes, it has been verified on bbc with https://*.live.bbctvapps.co.uk/ as origin pattern